src/source/quick-brick-epg-screen/QuickBrickEpgScreenVM.bs

```diff
@@ -383,15 +411,22 @@ class QuickBrickEpgScreenVM extends zapp.uib.ScreenVM
   end function
 
   function onLeftEdgeTargetTime()
-    dt = CreateObject("roDateTime")
-    now = dt.AsSeconds()
+    now = CreateObject("roDateTime").asSeconds()
     content = m.timeGrid.content
+    m.currentTimeGridLeftEdgeTime = m.TimeGrid.leftEdgeTargetTime
```

Comment:
split into a function, with a unit test.
m.currentTimeGridLeftEdgeTime = this.calculateLeftEdgeTime()